setup.py

```diff
@@ -22,24 +22,26 @@
 assert __version__
 
 # Build a list of all project modules, as well as supplementary files
-main_package = "spinn_front_end_common"
+main_packages = ("spinn_front_end_common", "tools")
```

Comment:
I would change the main package from "tools" to "spybug" and from tools import   is too generic

Reply:
spy implies only pynn.......